pymess/models/dialer.py

```diff
@@ -59,12 +66,7 @@ def clean_recipient(self):
 
     @property
     def failed(self):
-        return self.state == self.STATE.ERROR
-
-    class Meta(BaseMessage.Meta):
-        abstract = True
-        verbose_name = _('dialer message')
-        verbose_name_plural = _('dialer messages')
+        return self.state in {self.STATE.ERROR, self.STATE.ERROR_NOT_SENT}
```

Comment:
The question is what does this property imply ... in case of `ERROR` it does not mean it was not sent... I would only check ERROR_NOT_SENT to be consistent with behavior of other messages.